src/API/useInAppDownloadPreview.js

```diff
@@ -5,11 +5,16 @@ import {
 import { useOkapiKy } from '@folio/stripes/core';
 
 export const useInAppDownloadPreview = (id, capability) => {
```

Comment:
@uladislausamets I already fixed that in my previous PR. Please take a look on changes in master branch